pkgs/applications/graphics/zgv/switch.patch

```diff
@@ -0,0 +1,14 @@
+GCC complains
```

Comment:
If this is only to shut up a complaint, I think this would be more appropriate as an upstream patch. It doesn't really affect the build, does it?